src/services/git/Git.ts

```diff
@@ -19,28 +21,91 @@ export class Git {
     this.cache = cache;
   }
 
-  async listDirectory(path: string): Promise<(GitHubFile | GitHubDir)[]> {
-    return this.cache.getOrSet(this.contentCacheKey(path), async () => {
-      const params = GitHubUrlParser.getOwnerAndRepoName(this.repository.url);
-      const result = await this.gitHubClient.getRepoContent(params.owner, params.repoName, path);
-      if (isArray(result)) {
-        return result;
-      } else {
-        throw ErrorFactory.newInternalError(`${path} is not a directory`);
-      }
-    });
+  async exists(path: string): Promise<boolean> {
+    const result = await this.getRepoContent(await this.followSymLinks(path));
```

Comment:
Well, here, the target is GitHub. Symlinks work fine there. Since `FileSystemService.exists` does follow symlinks (though we do not test it anymore) and since GitHub's API doesn't, I have to implement it myself to make it work the same as `FileSystemService`.